### PR TITLE
core(fr): add more gatherers to config

### PR DESCRIPTION
**Pull request by connorjclark**
forgot this in #11993

ref #11313

### CODE REVIEW

File: lighthouse-core/fraggle-rock/config/default-config.js [hunk 1]
```diff
@@ -16,7 +16,24 @@ const defaultConfig = {
   navigations: [
     {
       id: 'default',
-      artifacts: ['Accessibility', 'ConsoleMessages'],
+      artifacts: [
+        'Accessibility',
+        'Appcache',
```
Comment (patrickhulce) [inline lighthouse-core/fraggle-rock/config/default-config.js:21]: they need to be defined in `artifacts` too, once that happens, the snapshot assertion for number of audits we compute should fail :)

Comment (connorjclark) [reply lighthouse-core/fraggle-rock/config/default-config.js:21]: ah ok.

btw i found this vscode plugin, super useful https://marketplace.visualstudio.com/items?itemName=wmaurer.change-case

File: lighthouse-core/test/fraggle-rock/api-test-pptr.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ describe('Fraggle Rock API', () => {
 
       const {auditResults, erroredAudits, failedAudits} = getAuditsBreakdown(lhr);
       // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
-      expect(auditResults.length).toMatchInlineSnapshot(`58`);
+      expect(auditResults.length).toMatchInlineSnapshot(`65`);
```
Comment (patrickhulce) [inline lighthouse-core/test/fraggle-rock/api-test-pptr.js:92]: 🎉 slowly but surely

**Review by patrickhulce — APPROVED** [review]

**Comment by patrickhulce** [comment]
image


this has been failing annoyingly often lately :/ I wonder if there's anything we can do

**Review by adamraine — APPROVED** [review]

**Pull request merged**